en/micro-integrator/docs/references/connectors/mongodb-connector/mongodb-connector-example.md

```diff
@@ -0,0 +1,257 @@
+# MongoDB Connector Example
+
+The MongoDB Connector can be used to perform CRUD operations in the local database as well as in MongoDB Atlas (cloud version of MongoDB).
+
+## What you&#39;ll build
+
+This example explains how to use MongoDB Connector to insert and find documents from MongoDB database.
+
+The sample API given below demonstrates how the MongoDB connector can be used to connect to the MongoDB Server and perform **insert many and find** operations on it.
+
+- `/insertmany`: The user sends the request payload which includes the connection information, collection name and the documents to be inserted. This request is sent to WSO2 EI by invoking the MongodbConnector API. It will insert the documents into the MongoDB database.
```

Comment:
```suggestion
- `/insertmany`: The user sends a request payload that includes the connection information, collection name, and the documents to be inserted. This request is sent to WSO2 EI by invoking the MongodbConnector API. This will insert the documents into the MongoDB database.
```